packages/core/lib/commands/debug/meta.js

```diff
@@ -35,6 +39,11 @@ module.exports = {
       OS.EOL +
       "                             [--compile-tests|--compile-all|--compile-none]",
     options: [
+      {
```

Comment:
The order of these controls the order of display, so I'd suggest moving this down... `--url` shouldn't be the very first thing, the transaction hash should be.  Somewhere after this but before the compile options, I think.